metadata.rb

```diff
@@ -24,7 +24,7 @@
 chef_version     '>= 12' if respond_to?(:chef_version)
 depends          'compat_resource'
 
-gem              'oneview-sdk' if respond_to?(:gem)
+gem              'oneview-sdk', '=1.0.0' if respond_to?(:gem)
```

Comment:
Gotcha. Ok, :+1:  from me. Do you know which resources are incompatible with 2.0.0?